### PR TITLE
fix: util get obfuscated

### PR DESCRIPTION
**Pull request by simboonlong**
- https://www.pivotaltracker.com/story/show/176797231

### CODE REVIEW

File: src/common/utils.test.ts [hunk 1]
```diff
@@ -64,15 +64,20 @@ const documentRopstenObfuscated: WrappedDocument<v2.OpenAttestationDocument> = {
...
   it("should return true where there is obfuscated data in v2 document", () => {
-    const { isOfuscated, obfuscated } = getObfuscated(documentRopstenObfuscated);
-    expect(isOfuscated).toBe(true);
-    expect(obfuscated?.length).toBe(1);
-    expect(obfuscated?.[0]).toBe("45c49a0e6efbde83c602cf6bbe4aa356d495feaf78a9a309cc1bad101f5c52f4");
+    expect(isObfuscated(documentRopstenObfuscated)).toBe(true);
+  });
+
+  it("should return undefined when there is no obfuscated data in v2 document", () => {
+    expect(getObfuscatedData(documentRopstenNotObfuscated)).toBe(undefined);
```
Comment (Nebulis) [inline src/common/utils.test.ts:75]: should we return an empty array instead ? I'm not fond of the undefined. wdyt ?

File: src/common/utils.ts [hunk 2]
```diff
@@ -11,27 +11,34 @@ export const getProvider = (options: VerificationBuilderOptions): providers.Prov
...
+  if (utils.isWrappedV3Document(document)) {
+    return !!document.proof.privacy?.obfuscated?.length;
+  }
+
+  if (utils.isWrappedV2Document(document)) {
+    return !!document.privacy?.obfuscatedData?.length;
+  }
+
+  throw new Error(
+    "Unsupported document type: Only can retrieve isObfuscated from wrapped OpenAttestation v2 & v3 documents."
```
Comment (Nebulis) [inline src/common/utils.ts:26]: suggestion: Can only check if there are obfuscated data from wrapped OpenAttestation v2 & v3 documents

File: src/common/utils.ts [hunk 3]
```diff
@@ -11,27 +11,34 @@ export const getProvider = (options: VerificationBuilderOptions): providers.Prov
...
-    return {
-      isOfuscated: !!document.privacy?.obfuscatedData?.length,
-      obfuscated: document.privacy?.obfuscatedData,
-    };
+    return document.privacy?.obfuscatedData;
   }
 
-  return { isOfuscated: false, obfuscated: undefined };
+  throw new Error(
+    "Unsupported document type: Only can retrieve obfuscated data from wrapped OpenAttestation v2 & v3 documents."
```
Comment (Nebulis) [inline src/common/utils.ts:42]: Can only ?

File: src/common/utils.test.ts (hunk 1 above)
Comment (simboonlong) [reply src/common/utils.test.ts]: no opinion on this, yeah i can return empty array

**Review by Nebulis — APPROVED** [review]

**Pull request merged**